common/app/assets/assets.scala

```diff
@@ -39,6 +39,34 @@ class Assets(base: String, mapResource: String, useHashedBundles: Boolean = Conf
 
 }
 
+// turns a readable CSS class into a list of rules in short form from the atomic css file
+class CssMap(mapResource: String, useCssMap: Boolean = Configuration.assets.useHashedBundles) extends Logging {
+
+  lazy val lookup: Map[String, List[String]] = Get(cssMap(mapResource))
+
+  def apply(path: String): String = {
+    if (useCssMap) {
```

Comment:
I would inline `useHashedBundles` here, at the location it's being used, for easier grep-ing. It makes it easier to understand where the system behaviour diverges for dev-asset loading. I doubt anyone will ever pass a second param to `CssMap`, and ... (see comment below)